app/workers/asset_manager_create_asset_worker.rb

```diff
@@ -3,15 +3,15 @@ class AssetManagerCreateAssetWorker < WorkerBase
 
   sidekiq_options queue: "asset_manager"
 
-  def perform(temporary_location, asset_params, draft = false, attachable_model_class = nil, attachable_model_id = nil, auth_bypass_ids = [])
+  def perform(temporary_location, asset_params, draft = false, attachable_model_class = nil, attachable_model_id = nil, auth_bypass_ids = [], access_limited_organisation_ids = [])
```

Comment:
Similar to my other comment: I'm not sure if this worker needs to accept a new `access_limited_organisation_ids` param if it didn't previously accept `access_limited` user IDs as a param 🤔